validation-test/stdlib/UnsafeBufferPointer.swift.gyb

```diff
@@ -361,6 +361,32 @@ UnsafeMutableBufferPointerTestSuite.test("withContiguous(Mutable)StorageIfAvaila
   expectEqual(result1, result2)
 }
 
+UnsafeMutableBufferPointerTestSuite.test("Slice.withContiguousStorageIfAvailable") {
+  guard #available(macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, *) else {
+    expectTrue(false)
```

Comment:
Oops, this slipped in from my local testing. I'll force push to get rid of it when the first test run completes,